pyccel/codegen/utilities.py

```diff
@@ -59,6 +60,10 @@ def construct_flags(compiler,
     if compiler == "mpif90":
         if debug:
             flags += " -fbounds-check"
+        if sys.platform == "win32":
+            mpiinc = os.environ["MSMPI_INC"].rstrip('\\')
+            mpilib = os.environ["MSMPI_LIB64"].rstrip('\\')
```

Comment:
Are these environment variables specific to a certain compiler?

Reply:
This is for MPI on Windows for any compiler, since Microsoft MPI is needed there.

Reply:
From some googling it seems that there is also an Intel MPI for windows. However if the user wishes to use this I think we can suppose that they will specify their own flags. In this case would we end up with multiple MPI routines specified?

If this isn't compiler specific then why is it in the `if compiler == "mpif90":`? Is this once again related to the default compiler? If the user specified their own compiler then won't the MPI flags be missed?

Reply:
Depending on which MPI library and compiler the user wants to use, I think that `mpif90` should point to the correct compiler. Now the question is: how do we know if the user wants to use the Intel MPI and compiler?

Reply:
Sorry, I just read in the comment below that the `mpif90` wrapper does not exist on Windows. I find that rather strange and inconvenient.,,

Reply:
For pyccel these are specified as options in the help. For epyccel they are in the kwargs